frontend/www/js/omegaup/login/signin.ts

```diff
@@ -83,13 +95,26 @@ OmegaUp.on('ready', () => {
               ui.error(T.loginPasswordTooShort);
               return;
             }
-
-            api.User.create({
-              username: username,
-              email: email,
-              password: password,
+            const request: {
+              username: string;
+              email?: string;
+              dateOfBirth: Date;
+              parentEmail?: string;
+              password: string;
+              recaptcha: string;
+            } = {
+              username,
+              dateOfBirth,
+              password,
               recaptcha: recaptchaResponse,
-            })
+            };
+            if (over13Checked) {
+              request.email = email;
+            } else {
+              request.parentEmail = parentEmail;
```

Comment:
Y acá sería de esta forma:
```suggestion
              request.parent_email = parentEmail;
```